src/IO/HDF5/ParallelHDF5IOHandler.cpp

```diff
@@ -74,6 +74,14 @@ ParallelHDF5IOHandlerImpl::ParallelHDF5IOHandlerImpl(AbstractIOHandler* handler,
     herr_t status;
     status = H5Pset_dxpl_mpio(m_datasetTransferProperty, xfer_mode);
 
+    auto const strByte = auxiliary::getEnvString( "OPENPMD_HDF5_ALIGNMENT", "0" );
```

Comment:
The default value is `1` according to the [docs](https://support.hdfgroup.org/HDF5/doc/RM/H5P/H5Pset_alignment.htm):
```suggestion
    auto const strByte = auxiliary::getEnvString( "OPENPMD_HDF5_ALIGNMENT", "1" );
```